### ISSUE TITLE
Nextcloud 21.0.2 - container keep rebooting

### ISSUE THREAD

**Issue by tamet83** [nextcloud/docker#1501]
Hello,

I just pulled the new image from Docker hub and something is not working.
Running the command `docker ps` I noticed that the container keep restarting.
I tried to look into nextcloud log but it is empty.

Could you please tell me what kind of information I need to provide you to debug the problem?

Many thanks.

**Comment by WhiteBahamut** [comment]
What is your arch? I noticed arm arch is missing in 21.0.2 images :(

**Comment by tamet83** [comment]
I use the `nexcloud` Image on an intel machine.
So I think it’s not this the problem. 
I don’t know what kind of log I should share.

**Comment by WhiteBahamut** [comment]
maybe related to #1500? Have you tried `21.0.1`?

**Comment by tamet83** [comment]
I updated from 21.0.1 to .2

**Comment by tamet83** [comment]
I solved the problem.
Last version of Nextcloud needed somehow check some files permissions.
Reading the docker log i noticed a problem with my custom log folder.
I solved with following steps:
1. Stopping and deleting the container
2. Comment the line with custom log folder in my docker compose
3. Rebuild the container and let doing the check.
4. Stopping and deleting again the container
5. Reactivate the custom log folder
6. Rebuild again the container

Thank anyway for the help :)
